app/src/main/res/layout/fragment_home_news.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#F00070"> <!-- TODO: hardcoded because is for testing -->
```

Comment:
Solve this harcoded testin dimen

Reply:
This is hardcoded because is the color of the background used for testing that news and profile pages are changing from one to another, will be removed on the next PR where I will create the layout for each pages. What do you think?

Reply:
Ok, I understand. We are going to leave it and change it in future